.github/workflows/plugins-benchmark-pr.yml

```diff
@@ -7,7 +7,30 @@ on:
         type: string
         default: benchmark
         required: false
-
+      source-repo: 
```

Comment:
Since we are fixing the wording, could we improve the step's `name`?

"Run the benchmark on the main branch. It is the minimum TARGET one."
"Run the benchmark on the pr-branch to spot evident regressions."

Reply:
I think I should just change everything to GitHub's PR terminology. You merge from the "head" into the "base".

So then
`source-repo` -> `head-repo`
`target-sha` -> `base-sha`
`PR-BENCH-18` -> `HEAD-BENCH-18`
`TARGET-BENCH-18` -> `BASE-BENCH-18`

etc

wdyt @Eomm ?

Reply:
To be honest that teminology confuses me.

Reply:
I like `PR-*` and `BASE-*` (instead of target) naming - it does not make me think "what is this?"

Do you agree @Uzlopak

Reply:
Same, tbh. Do we want to go with
`source-repo`/`source-sha`/`source-ref`/`SOURCE-BENCH-18`/...
and
`target-repo`/`target-sha`/`target-ref`/`TARGET-BENCH-18`/...

And I'll update the step names from:
`- name: Run benchmark`
to
`- name: Run benchmark (${{ inputs.source-repo }}@${{ inputs.source-ref }})`

Reply:
Sorry, left my comment before I saw yours.

I am also good with PR instead of SOURCE, but think the inputs should match, so they would go to `pr-repo`, etc.

Reply:
I like the proposal of Eomm, because it is clear what the PR is.